packages/beacon-node/src/metrics/metrics/beacon.ts

```diff
@@ -348,6 +348,10 @@ export function createBeaconMetrics(register: RegistryMetricCreator) {
         name: "beacon_data_column_sidecar_gossip_verification_seconds",
         help: "Full runtime of data column sidecars gossip verification",
       }),
+      dataColumnSideCarComputationTime: register.histogram({
+        name: "beacon_data_column_sidecar_computation_seconds",
+        help: "Time taken to compute data column sidecar, including cells, proofs and inclusion proof",
```

Comment:
Not hugely important, but we're no longer computing proofs in the CL, and since this calculates multiple sidecars, might be more accurate to call the metric:

```suggestion
        name: "beacon_data_column_sidecars_computation_seconds",
        help: "Time taken to compute data column sidecars, including cells and inclusion proof",
```

Also, the time taken to complete this will vary based on the number of blobs in a block, but this can probably be cross-referenced with blobs-per-block somewhere.

Reply:
https://github.com/ethereum/beacon-metrics/pull/14 seems to be outdated as per latest peerdas spec

Reply:
Changing the `help` text is feasible, but renaming the metric from `sidecar` to `sidecars` is challenging. The name should follow the specs and this metric is already implemented in some clients. This change would require updates to the specification, the spec dashboard, and client implementations. Is it worth the effort?

Reply:
If it's going to be a lot of work to change, personally I'd be in favor of merging now and cleaning it up later once peerDAS has settled a bit. Just to double-check, the metric is being used to time computation of multiple data column sidecars in other clients as well?

Reply:
I agree that shouldn't change metric name for now and follow spec, this can be discussed separately and adapted later in Lodestar if needed